package.json

```diff
@@ -8,10 +8,9 @@
   },
   "scripts": {
     "clean": "rm -rf build/ .parcel-cache",
-    "start": "REACT_APP_META_CSP= parcel --port 3000 --target web --dist-dir build",
     "parcel:prod": "parcel build --target web --dist-dir build",
     "clean:ext": "rm -rf build-ext/ .parcel-cache",
-    "start:ext": "HMR_PORT=2222 EXTENSION_CSP=`yarn --silent print-extension-csp` parcel --host localhost --target ext --dist-dir build-ext",
+    "start": "REACT_APP_META_CSP= HMR_PORT=2222 EXTENSION_CSP=`yarn --silent print-extension-csp` parcel --host localhost --port 3000 --dist-dir build-dev",
```

Comment:
How about we add `REACT_APP_BYPASS_LOCAL=1` such that `yarn start` works out of the box by default without local node?

Reply:
edit: REACT_APP_LOCALNET=